soc/arm/nxp_kinetis/k6x/dts_fixup.h

```diff
@@ -122,4 +122,17 @@
 #define CONFIG_USBD_KINETIS_BASE_ADDRESS	NXP_KINETIS_USBD_40072000_BASE_ADDRESS
 #define CONFIG_USBD_KINETIS_NUM_BIDIR_EP	NXP_KINETIS_USBD_40072000_NUM_BIDIR_ENDPOINTS
 
+#define CONFIG_ETH_MCUX_0_NAME		ETH_LABEL
+
+#define CONFIG_ETH_MCUX_0_MAC3		NXP_KINETIS_ETHERNET_400C0004_LOCAL_MAC_ADDRESS_3
+#define CONFIG_ETH_MCUX_0_MAC4		NXP_KINETIS_ETHERNET_400C0004_LOCAL_MAC_ADDRESS_4
+#define CONFIG_ETH_MCUX_0_MAC5		NXP_KINETIS_ETHERNET_400C0004_LOCAL_MAC_ADDRESS_5
+
+/* IRQs */
+#define IRQ_ETH_IEEE1588_TMR		NXP_KINETIS_ETHERNET_400C0004_IRQ_ERR_MISC
```

Comment:
Can you make these CONFIG_IRQ... (we need to update any references in the driver)

Reply:
Hey @galak  I've made the change.